bundle/manifests/backstage-operator.clusterserviceversion.yaml

```
@@ -21,7 +21,7 @@ metadata:
         }
       ]
     capabilities: Seamless Upgrades
-    createdAt: "2024-04-02T12:53:25Z"
+    createdAt: "2024-04-02T13:59:53Z"
```

```suggestion
    createdAt: "2024-04-02T12:53:25Z"
```

If the `createdAt` date is the only change here, I think we can skip it.